docs/layouts/shortcodes/endpoint-clarification.html

```diff
@@ -0,0 +1,37 @@
+
+{{ $data := index .Site.Data.registry ( .Get 0 )}}
+{{ $link := ""}}
+{{ $multiples := "" }}
+{{ if or ( eq (.Get 0) "solr") (eq (.Get 0) "mariadb") }}
+  {{ $link = (.Get 1) }}
+  {{ $multiples = (.Get 2)}}
+{{ end }}
+
+<div class="alert alert-primary" role="alert">
```

Comment:
This code looks like an unmaintainable mess.  I can't even read what it's saying from the source.  What is it...?

At minimum this file needs documentation itself, but just glancing at it I don't even know what it's doing.